packages/core/framework/src/build-tools/compiler.ts

```diff
@@ -163,6 +163,7 @@ export class Compiler {
    * for MedusaJS loaders during development
    */
   async #createPluginOptionsFile() {
+    await mkdir(path.dirname(this.#pluginOptionsPath), { recursive: true })
     await writeFile(
```

Comment:
Q: If we use our FileSystem util, it would do it automatically right?

Reply:
Good point @adrien2p

Reply:
Something we want to do?